tests/arp/test_wr_arp.py

```diff
@@ -190,6 +190,21 @@ def setupRouteToPtfhostFixture(self, duthosts, rand_one_dut_hostname, ptfhost):
         yield
         self.teardownRouteToPtfhost(duthost, route, ptfIp, gwIp)
 
+    @pytest.fixture(scope='class', autouse=True)
+    def warmRebootSystemFlag(self, duthost):
+        """
+            Sets warm-reboot system flag to false after test. This class-scope fixture runs once before test start
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        yield
+        logger.info('Setting warm-reboot system flag to false')
+        duthost.shell(cmd='sonic-db-cli STATE_DB hset "WARM_RESTART_ENABLE_TABLE|system" enable false')
```

Comment:
This is not safe. I think the safe way to go is to wait_until the flag is cleared for 5 minutes, and clear the flag upon timeout.